SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -3531,6 +3533,133 @@ bool USpatialNetDriver::HasTimedOut(const float Interval, uint64& TimeStamp)
 	return false;
 }
 
+void USpatialNetDriver::ServerReplicateActors_BuildConsiderList(TArray<FNetworkObjectInfo*>& OutConsiderList, const float ServerTickTime)
+{
+	TArray<FNetworkObjectInfo*> TmpConsiderList;
```

Comment:
Because of the way this code is structured, and the need to put 'HasAuthority' here rather than in Super, it means we have to allocate and copy 'considerlist' arrays twice (rather than once in the original code). Not sure how much of a performance issue this is however.

Reply:
This is a very good point and I have put back the original code in Super so that this extra code only runs in debug mode.